install.sh

```diff
@@ -841,19 +853,47 @@ create_user() {
 # re-evaluate args to include env command
 eval set -- $(escape "${INSTALL_RKE2_EXEC}") $(quote "$@")
 
+# setup_rke2_user creates the rke2 user and group, home
+# directory, and sets necessary ownership.
+setup_rke2_user() {
+    mkdir -p "${BASE_DIR}"
+    create_user "$1" "RKE2 Service User"
+    chown -R "$1":"$1" "$(dirname ${BASE_DIR})"
+    USING_RKE2_USER=1
+}
+
+# setup_etcd_user creates the etcd user, provides a description
+# and adds it to the rke2 group if it exists.
+setup_etcd_user() {
+    create_user "$1" "ETCD Service User"
+    if [ "$(id -u "rke2" 2>/dev/null)" = 1 ]; then
+        usermod -a -G "${INSTALL_RKE2_USER}" "${INSTALL_RKE2_ETCD_USER}"
+    fi
+    USING_ETCD_USER=1
+}
+
+# update_kernel_params adjusts the necessary kernel parameters
+# to allow RKE2 to run in CIS mode.
+update_kernel_params() {
+    sysctl -w vm.panic_on_oom=0
+    sysctl -w kernel.panic=10
+    sysctl -w kernel.panic_on_oops=1
+    sysctl -w kernel.keys.root_maxbytes=25000000
```

Comment:
These persist until the next reboot. Should we not attempt to drop-in (and apply) an `/etc/sysctl.d` entry?

Reply:
I think that'd actually be preferred. I'll do that. Thanks for bringing that up!